python/lib/adaguc/CGIRunner.py

```diff
@@ -15,14 +16,18 @@
 import re
 #from numba import jit
 
+# TODO: This limits the number of adaguc process calls per fastapi worker.
+#   Do we want this?
+#   Make configurable
+sem = asyncio.Semaphore(4)
```

Comment:
Can we make this configurable via an environment var or something?

Reply:
Also in Docker/start.sh, the number of gunicorn workers is still 4.

Could this now be set to 1?